### PR TITLE
Increase coverage to 90%

### LINKED ISSUE #473: Increase code coverage to at least 90%
[Coverage Status](https://coveralls.io/github/gnosis/safe-transaction-service?branch=master)

### PR DESCRIPTION
**Pull request by Uxio0**
### What was wrong?

Closes #473

### How was it fixed?

@gnosis/safe-services

### CODE REVIEW

**Comment by Uxio0** [comment]
Getting closer to 90%

**Review by fmrsabino — CHANGES_REQUESTED** [review]

File: .coveragerc [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+    # Don't complain if tests don't hit defensive assertion code:
+    raise AssertionError
+    raise NotImplementedError
+
+    # Don't complain if non-runnable code isn't run:
+    if 0:
+    if __name__ == .__main__.:
+
+    # Don't complain about abstract methods, they aren't run:
+    @(abc\.)?abstractmethod
```
[the remark this answers is not in the document]
Comment (Uxio0) [reply .coveragerc:15]: Good points. I picked the example from https://coverage.readthedocs.io/en/6.1.1/config.html#syntax

File: .coveragerc [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+exclude_lines =
+    # Have to re-enable the standard pragma
+    pragma: no cover
+
+    # Don't complain if tests don't hit defensive assertion code:
+    raise AssertionError
+    raise NotImplementedError
+
+    # Don't complain if non-runnable code isn't run:
+    if 0:
```
[the remark this answers is not in the document]
Comment (Uxio0) [reply .coveragerc:11]: That was on the example configuration :)

File: .coveragerc [hunk 3]
```diff
@@ -1,3 +1,19 @@
+[report]
+exclude_lines =
+    # Have to re-enable the standard pragma
+    pragma: no cover
```
[the remark this answers is not in the document]
Comment (Uxio0) [reply .coveragerc:4]: To manually exclude stuff (for example, Celery configuration)

File: .coveragerc [hunk 4]
```diff
@@ -1,3 +1,19 @@
...
+
+    # Don't complain if tests don't hit defensive assertion code:
+    raise AssertionError
+    raise NotImplementedError
```
[the remark this answers is not in the document]
Comment (Uxio0) [reply .coveragerc:8]: Maybe is broad on `AssertionError`, but `NotImplemented` error are only for not implemented abstract methods, so it should be always excluded

[the remark this answers is not in the document]
Comment (Uxio0) [reply .coveragerc:8]: I don't want to go marking abstract methods one by one

**Review by fmrsabino — APPROVED** [review]

**Pull request merged**